plugins/azure_sentinel/help.md

```diff
@@ -511,190 +651,105 @@ _This plugin does not contain any triggers._
-|System Data|SystemData|False|Azure Resource Manager metadata containing createdBy and modifiedBy information|
-|Tactics|[]string|False|The tactics of the alert|
-|Title|date|False|The time the alert was generated|
-|Title|string|False|Azure resource type|
-|Vendor Name|string|False|The name of the vendor that raised the alert|
-
-#### ConfidenceReasons
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|Reason|string|False|Reason's description|
-|Reason Type|string|False|The reason's type (category)|
-
 #### CreatedByType
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
 |Application|string|False|Application|
 |Key|string|False|Description|
-|Managed Indentity|string|False|Managed identity|
+|Managed Indetity|string|False|Managed Identity|
 |User|string|False|User|
 
-#### Entity
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|Azure resource ID|string|False|Azure resource ID|
-|Kind|string|False|The kind of the entity|
-|Name|string|False|Azure resource name|
-|Properties|object|False|Entity properties|
-
-#### HuntingBookmark
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|ID|string|False|Resource ID|
-|Kind|string|False|The kind of the entity|
-|Name|string|False|Resource name|
-|Properties|HuntingBookmarkProperties|False|Hunting bookmark properties|
-|System Data|SystemData|False|Azure Resource Manager metadata containing createdBy and modifiedBy information|
-|Type|string|False|Azure resource type|
-
-#### HuntingBookmarkProperties
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|Additional Data|object|False|Custom fields that should be part of the entity and will be presented to the user|
-|Created|date|False|The time the bookmark was created|
-|Created By|UserInfo|False|Describes a user that created the bookmark|
-|Display Name|string|False|The display name of the bookmark|
-|Event Time|date|False|The time of the event|
-|Friendly Name|string|False|The graph item display name which is a short human-readable description of the graph item instance|
-|Incident Info|object|False|Describes an incident that relates to bookmark|
-|Labels|[]string|False|List of labels relevant to this bookmark|
-|Notes|string|False|The notes of the bookmark|
-|Query|string|False|The query of the bookmark|
-|Query Result|string|False|The query result of the bookmark|
-|Updated|date|False|The last time the bookmark was updated|
-|Updated By|UserInfo|False|Describes a user that updated the bookmark|
-
 #### Incident
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|Etag|string|False|Etag|
-|ID|string|False|Full incident ID|
-|Name|string|False|Incident name - short ID|
-|Incident Properties|IncidentProperties|False|Incident properties object|
-|Type|string|False|Incident type|
+|etag|string|False|None|
+|Full Incident ID|string|False|Full incident ID|
```

Comment:
Some of the name fields in multiple output types are the descriptions instead, can these be changed back to the correct name?